tests/sceneManager_test.cpp

```diff
@@ -57,4 +57,31 @@ TEST_F(SceneManagerTest, EnergyCollectorPositionsCheck)
 
     Vec3 referenceCollisionPoint3(0, manager.simulatiorRadius() / 2 - manager.collectorRadius(), 0);
     ASSERT_EQ(hitData.collisionPoint, referenceCollisionPoint3) << "Invalid hit point from: " << againstXAxis;
+
```

Comment:
I am going to add a comment here because I cannot do them above:

1. Line 49, ray is named incorrectly
2. Line 55, ray is named incorrectly
3. Remove those constant "Invalid hit" << ... because they only clutter the code, and do not actually print useful stuff. Note that if you named the referencePoint3 correctly, you would immediately know which assert failed.